chainstate/src/detail/tx_verification_strategy/default_strategy.rs

```diff
@@ -50,11 +51,12 @@ impl TransactionVerificationStrategy for DefaultTransactionVerificationStrategy
         verifier_config: TransactionVerifierConfig,
         block_index: &'a BlockIndex,
         block: &WithId<Block>,
-    ) -> Result<TransactionVerifier<'a, S>, BlockError>
+    ) -> Result<TransactionVerifier<'a, S, P>, BlockError>
     where
         H: BlockIndexHandle,
         S: TransactionVerifierStorageRef,
-        M: Fn(&'a S, &'a ChainConfig, TransactionVerifierConfig) -> TransactionVerifier<'a, S>,
+        P: UtxosView,
```

Comment:
minor thing but maybe replace P with UV. I guess P stands for parent but in case of verifier it's not that obvious

Reply:
Done.